providers/openstack/alpha/1-27/cluster-class/templates/cluster-class.yaml

```diff
@@ -158,7 +158,82 @@ spec:
          default: ""
          example: "capi-keypair"
          description: "The ssh key to inject in the nodes."
+   - name: certSANs
+     required: false
+     schema:
+       openAPIV3Schema:
+         type: array
+         default: []
+         example: ["mydomain.example"]
+         description: "CertSANs sets extra Subject Alternative Names for the API Server signing cert."
+         items:
+           type: string
+   - name: oidc_config
+     required: false
+     schema:
+       openAPIV3Schema:
+         type: object
+         properties:
+           client_id:
+             type: string
+             example: "kubectl"
+             description: "A client id that all tokens must be issued for."
+           issuer_url:
+             type: string
+             example: "https://dex.k8s.scs.community"
+             description: "URL of the provider that allows the API server to
```

Comment:
is that valid yaml? If, thats fine, if not you can use the multiline syntax

```yaml
key: |
  This is a multiline strings
  and yaml knows because of the indentation and the
  starting pipe
```